packages/truffle-contract/test/util.js

```diff
@@ -20,18 +21,37 @@ var util = {
 
   // Compiles and instantiates (our friend) Example.sol
   createExample: function() {
+    return util._createContractInstance(
```

Comment:
I just followed in the footsteps of `createExample` here - refactoring the logic of that out to `_createContractInstance`